bson/doc.go

```diff
@@ -86,14 +89,14 @@
 //
 //     1. omitempty: If the omitempty struct tag is specified on a field, the field will not be marshalled if it is set to
 //     the zero value. By default, a struct field is only considered empty if the field's type implements the Zeroer
-//     interface and the IsZero method returns true. If the field's type does not implement Zeroer, the field will be
-//     marshalled as an embedded document.
+//     interface and the IsZero method returns true. Struct fields of types that do not implement Zeroer will be
+//     marshalled as embedded documents.
```

Comment:
Struct fields of types that do implement Zeroer are also marshalled as embedded document (when IsZero is false).  Do you mean something like:

    Struct fields of types that do not implement Zeroer are _always_ marshalled as embedded documents

?

Reply:
done.